R/def_minerals.R

```diff
@@ -9,35 +9,39 @@
 #' @param mineral_type \code{character}. The type of mineral (e.g.,
 #'   "feldspar") to return a definition for.
 #' @param element \code{character}. The chemical element abbreviation of a
-#'   mineral to return a definition for (e.g., "CaCO3"). Note: abbreviations
+#'   mineral (e.g., "CaCO3") to return a definition for. Note: abbreviations
 #'   are case sensitive.
 #'
 #' @return A \code{data.frame} containing the following columns:
 #' \itemize{
-#'   \item \code{mineral_id}: The unique identification number of the mineral.
+#'   \item \code{mineral_id}: The unique identification number(s) of the
+#'    mineral(s).
 #'   \item \code{mineral}: The name of the mineral.
 #'   \item \code{mineral_type}: The name of the type of mineral.
-#'   \item \code{hardness_min}: The minimum hardness value (Moh's hardness
-#'   scale) of the mineral.
-#'   \item \code{hardness_max}: The maximum hardness value (Moh's hardness
-#'   scale) of the mineral.
-#'   \item \code{mineral_color}: The color description of the mineral.
-#'   \item \code{lustre}: The lustre description of the mineral.
-#'   \item \code{crystal_form}: The crystal form of the mineral.
 #'   \item \code{formula}: The chemical formula of the mineral.
 #'   \item \code{formula_tags}: The chemical formula of the mineral with
 #'   appropriate sub/superscript html tags.
```

Comment:
Will made the point that we should indent when text flows overs more than one line to ease reading, e.g.:

```suggestion
#'     appropriate sub/superscript html tags.
```

Reply:
Sorry, I keep forgetting :)